### PR TITLE
My second homework

### PR DESCRIPTION
**Pull request by dhanur350**
Please Check the homework , do find if any mistakes I've done

### CODE REVIEW

File: 1_base_table.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	<meta charset='utf-8'>
+	<meta http-equiv='X-UA-Compatible' content='IE=edge'>
+	<meta name='viewport' content='width=device-width, initial-scale=1'>
+	<link rel="stylesheet" href="minimal.css"> 
+</head>
+<body>
+
+	<h1>My HomeTask</h1>
+	<h2>Dhanur</h2>
+	<u style="color: blue;">Link to my portfolio</u> <!--Should be <a> tag-->
```
Comment (igniteacademy) [inline 1_base_table.html:13]: Why we using <u> tag here ?
Definition of you tag: The <u> tag represents some text that is unarticulated and styled differently from normal text.
And this is supposed to be linke to your portfolio. PLease use <a> tag

File: 1_base_table.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	
+	<h3>Plan</h3>
+	<main>	
+	<ol> <!--Use tab or two spaces for styling code--> <!--I didn't get it what you are trying to say here???-->
```
Comment (igniteacademy) [inline 1_base_table.html:17]: Code style: Missing tab

File: 1_base_table.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+	<li>Javascript</li>
```
Comment (igniteacademy) [inline 1_base_table.html:18]: Code style: Missing tab

File: 1_base_table.html [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+		<li>Types</li>
+		<ul>
+			<li>String</li>
+			<li>Number</li>
+			<li>Boolean</li>
+		</ul>
+		</ol>
+		<li>CSS</li>
+		<li>HTML</li>
+		</ol>
```
Comment (igniteacademy) [inline 1_base_table.html:30]: Code style: end tag </ol> should be on the same level as open tag <ol>

File: 1_base_table.html [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+	
+		<hr>	 <!--I think you missued the tag. Could be <section> instead-->
+		<h1>My TimeTable</h1>
+		<table>	
+		<thead>
```
Comment (igniteacademy) [inline 1_base_table.html:35]: Code style: Missing tab

File: 1_base_table.html [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+		<tr>
+		<th colspan="2">Period</th>
```
Comment (igniteacademy) [inline 1_base_table.html:37]: Code style: Missing tab

File: 1_base_table.html [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+		<th>Tue</th>
+		<th>Wed</th>
+		<th>Thu</th>
+		<th>Fri</th>
+		<th>Sat</th>
+		<th>Sun</th>
+		</tr>
+		</thead>
+		<tbody>
+		<tr>
```
Comment (igniteacademy) [inline 1_base_table.html:48]: Code style: Missing tab

File: 1_base_table.html [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+		<th colspan="1" rowspan="2" scope="rowgroup">2nd Period</th>
+		<th>First Term</th>
+		<td>CSS</td>
+		<td>CSS</td>
+		<td></td>
+		<td>HTML</td>
+		<td>HTML</td>
+		<td></td>
+		<td></td>
+		</tr>
```
Comment (igniteacademy) [inline 1_base_table.html:68]: Code style: Close tag should be on the same level as open tag

File: 1_base_table.html [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+		<tr>
```
Comment (igniteacademy) [inline 1_base_table.html:69]: again code style

File: 1_base_table.html [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+		</tbody>
+		<tfoot>
+
+		</tfoot>
+		</table>	
+	<hr>
+
+    <div style="white-space: pre-line;">
+	<img style="width: 350px; height: 200px;" src="portfolio.png">
+	</div>
```
Comment (igniteacademy) [inline 1_base_table.html:108]: again code style: close tag should be the same level as open tag

File: 1_base_table.html [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+	<p> 202X My TIme Table</p>		
+	</main>
```
Comment (igniteacademy) [inline 1_base_table.html:110]: again code style: close tag should be the same level as open tag

File: 1_base_table.html [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+
+</body>
```
Comment (igniteacademy) [inline 1_base_table.html:112]: again code style: close tag should be the same level as open tag

**Review by igniteacademy — CHANGES_REQUESTED** [review]

File: 1_base_table.html (hunk 1 above)
Comment (dhanur350) [reply 1_base_table.html:13]: Mam here in the homework you did underline in the sentence so did I..

Comment (igniteacademy) [reply 1_base_table.html:13]: I didnt get what do you mean by underline. U tag is for styling. And in homework you have link to your protfolio so you can click it and go to any other website.

**Pull request merged**